docker/config/local_dev.env

```diff
@@ -105,7 +105,6 @@ OVERVIEW_VERSION_URLS=http://localhost:8000/__version__
 
 # crontabber
 # ----------
-crontabber.class-DependencySecurityCheckCronApp.nsp_path=/webapp-frontend-deps/node_modules/.bin/nsp
```

Comment:
We don't need this anymore since we're using `npm` which is installed into the Docker container at `/usr/bin/npm`.